src/scripts/popUp.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable import/no-cycle */
+
+import eventsPopUp from './listener.js';
+import showComments from './showComments.js';
+import { newSession } from './calls.js';
+
+export default function popUpAnime(e) {
+  const index = e.target.parentNode.id;
+  newSession.get('https://api.jikan.moe/v3/season/2021/summer')
+    .then((data) => {
+      const apiAnime = data.anime;
+      const body = document.querySelector('.popup');
+      const background = document.createElement('div');
+      const container = document.createElement('div');
+      background.classList.add('background');
+      container.classList.add('popup_container');
+      container.innerHTML = `
```

Comment:
[Optional]
`innerHTML` can cause security issues if you use it with data you don't have control over.  (Data coming through API) 
It is suggested to use `textContent` (or sanitize the data) for printing data that you get from a 3rd party.